imgui-SFML.cpp

```diff
@@ -673,18 +671,16 @@ void Image(const sf::RenderTexture& texture, const sf::Vector2f& size, const sf:
 /////////////// Image Overloads for sf::Sprite
 
 void Image(const sf::Sprite& sprite, const sf::Color& tintColor, const sf::Color& borderColor) {
-    const sf::FloatRect bounds = sprite.getGlobalBounds();
-    Image(sprite, sf::Vector2f(bounds.width, bounds.height), tintColor, borderColor);
+    Image(sprite, sprite.getGlobalBounds().size, tintColor, borderColor);
 }
 
 void Image(const sf::Sprite& sprite, const sf::Vector2f& size, const sf::Color& tintColor,
            const sf::Color& borderColor) {
     const sf::Texture& texture = sprite.getTexture();
     const sf::Vector2f textureSize(texture.getSize());
     const sf::FloatRect textureRect(sprite.getTextureRect());
-    const ImVec2 uv0(textureRect.left / textureSize.x, textureRect.top / textureSize.y);
-    const ImVec2 uv1((textureRect.left + textureRect.width) / textureSize.x,
-                     (textureRect.top + textureRect.height) / textureSize.y);
+    const ImVec2 uv0(textureRect.position.cwiseDiv(textureSize));
+    const ImVec2 uv1((textureRect.position + textureRect.size).cwiseDiv(textureSize));
```

Comment:
https://github.com/SFML/imgui-sfml/pull/268

Using these conversion constructors may cause problems

Reply:
Updated the PR and tested it with `IM_VEC2_CLASS_EXTRA` and `IM_VEC4_CLASS_EXTRA` undefined, it compiles now.